web/sites/default/settings.php

```diff
@@ -794,3 +794,5 @@
 if (file_exists($app_root . '/' . $site_path . '/settings.local.php')) {
   include $app_root . '/' . $site_path . '/settings.local.php';
 }
+
+$settings['skip_permissions_hardening'] = TRUE;
```

Comment:
This should be put above the `settings.local.php` stuff (cf. “Keep this code block at the end of this file to take full effect.”). We should be able to override anything in `settings.local.php` – even the `skip_permissions_hardening` setting.